client/post-editor/editor-more-options/copy-post.jsx

```diff
@@ -66,6 +66,11 @@ class EditorMoreOptionsCopyPost extends Component {
 
 	render() {
 		const { translate, siteId } = this.props;
+
+		if ( ! siteId ) {
```

Comment:
Or we could update `<PostSelector />` to accommodate empty `siteId`, just rendering an empty display (perhaps inferred as loading, showing placeholder).